delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/client/BlazegraphClient.scala

```diff
@@ -110,16 +127,38 @@ class BlazegraphClient(
 
 object BlazegraphClient {
 
+  /**
+    * Header defining Blazegraph query timeout
+    *
+    * @param duration max query duration
+    */
+  final class `X-BIGDATA-MAX-QUERY-MILLIS`(duration: FiniteDuration)
```

Comment:
Do we need all of this in order to create a custom Http Header? It seems very verbose and unnecessary. I think you can create a raw header with a single line in akka http